.github/workflows/main-ci.yml

```diff
@@ -0,0 +1,122 @@
+name: CI/CD Pipeline
+
+on:
+  push:
+    branches:
+      - feature/18-ci-cd-build-pipeline-with-github-actions
```

Comment:
@vasylboliuk that's too specific. It won't work for any other MR branches. Is there a wildcard or something like that in github actions like in gitlab CI/CD?